stripe/src/main/java/com/stripe/android/LoggingUtils.java

```diff
@@ -21,32 +20,17 @@
  * Util class to create logging items, which are fed as {@link java.util.Map Map} objects in
  * query parameters to our server.
  */
-public class LoggingUtils {
-
-    @Retention(RetentionPolicy.SOURCE)
-    @StringDef({
-            ANDROID_PAY_TOKEN,
-            CARD_WIDGET_TOKEN
-    })
-    public @interface LoggingToken { }
-    public static final String ANDROID_PAY_TOKEN = "AndroidPay";
```

Comment:
I guess this move will mean Android Pay will need it's own equivalent of a LoggingUtils class?